plugin-tester-scala/pom.xml

```diff
@@ -39,13 +54,50 @@
           <language>SCALA</language>
           <protoPath>src/main/protobuf</protoPath>
         </configuration>
+        <executions>
+          <execution>
+            <goals>
+              <goal>generate</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>2.5.1</version>
+        <executions>
+          <execution>
+            <id>getClasspathFilenames</id>
+            <goals>
+              <!-- provides the jars of the classpath as properties inside of maven
+                   so that we can refer to one of the jars in the exec plugin config below -->
+              <goal>properties</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>1.6.0</version>
+        <configuration>
+          <executable>java</executable>
+          <arguments>
+            <argument>-javaagent:${org.mortbay.jetty.alpn:jetty-alpn-agent:jar}</argument>
+            <argument>-classpath</argument>
+            <classpath />
+            <argument>example.myapp.helloworld.GreeterServer</argument>
+          </arguments>
+        </configuration>
       </plugin>
     </plugins>
   </build>
   <properties>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
     <grpc.version>1.12.0</grpc.version>
+    <akka.grpc.project.version>ce349193+20180621-1030</akka.grpc.project.version>
```

Comment:
Should we remove this line? (when testing you can set it from the command line with `-Dakka.grpc.project.version=ce349193+20180621-1030`)

Reply:
With the comment up top I was thinking it is nicer to have it here as a placeholder, but sure, I'll remove it.